docs/examples/ai_ml_inferencing.md

```diff
@@ -1 +1,3 @@
 # AI/ML/Inferencing
+
+To demonstrate some of the potential usages of ROCm for AI/ML/DL/Inferencing we provide a detailed example of a ROCm implementation of Inception v3 using the PyTorch framework.
```

Comment:
A link to the example should be included

Reply:
Link from into to Inception V3 with PyTorch example was added